Agreements/External/MAW_DUBN.txt

```diff
@@ -1,3 +1,3 @@
-FIR_COPX:*:*:RAMOX:EIDW:*:Scottish PC West:DUBN:*:20000:RAMOX
-FIR_COPX:*:*:SITKU:EIDW:*:Scottish PC West:DUBN:*:20000:SITKU
+FIR_COPX:*:*:RAMOX:EIDW:*:Scottish PC West:DUBN:*:20000:|RAMOX
+FIR_COPX:*:*:SITKU:EIDW:*:Scottish PC West:DUBN:*:20000:|SITKU
 FIR_COPX:*:*:GINIS:EIDW:*:Scottish PC West:DUBN:*:16000:GINIS
```

Comment:
Any idea what this agreement is @cpawley? I can't see that it exists anywhere (anymore)

Reply:
Yup - can't find any reference to it

Reply:
I think it's old and can be removed